app/javascript/submission_handling/renewal.js

```diff
@@ -0,0 +1,35 @@
+import { allChecked, findForm, responseFromRails, spinner } from './shared'
```

Comment:
So I see you named this `renewal.js` instead of `checkout.js` - I think I expected everything to be checkouts since you got rid of the renewal controller 🤔

Reply:
hmmm, I was thinking to use checkouts for display checkouts async js 🤔 maybe `renew_checkouts` for this and `view_checkouts` for display?

Reply:
yep I like that. I was talking about the filename to be clear... should it still be `renewal.js` or `renew_checkouts.js`? The latter right?

Reply:
yup, the filename, I added a commit for the change, could you check if looks good?

Reply:
thanks that makes more sense to me